### DISCUSSION TITLE
Rethinking the Find & Select pane

### DISCUSSION THREAD

**Discussion by pastewka** [ContactEngineering/topobank#753] [Ideas]
I would like to start a discussion on the "Find & Select" pane. We are adding more and more information to it and I think the tabular view that we have right now no longer works. For inspiration, here are how other sites are doing it (data and journals):

## Figshare:
![image](https://user-images.githubusercontent.com/2530391/138554503-94448ab5-e3b5-444c-be31-d93657eb6d20.png)

## Zenodo:
![image](https://user-images.githubusercontent.com/2530391/138554525-071bb19f-f642-4581-a18f-58988204f86a.png)

## Springer:
![image](https://user-images.githubusercontent.com/2530391/138554541-bc817426-80a6-43be-bf59-da4607d43461.png)

## Elsevier:
![image](https://user-images.githubusercontent.com/2530391/138554581-7761641c-0e13-4ec3-b7a8-fe7a88faf449.png)

I think I like the Zenodo way. I would fit quite well what we have. To the left of the author list I would have a little toggle button; if the toggle is off then the data is unpublished and we don't show authors (just the term "unpublished dataset" or similar).

@tdbjacobs @mcrot - any thoughts?

This discussion is also related to issues #538 #750

**Comment by tdbjacobs** [comment]
I like these options.

But I don’t think there’s much benefit in having graphics in addition to
text. Most surfaces are not readily identifiable from their topographical.

Tevis

On Sat, Oct 23, 2021 at 7:47 AM Lars Pastewka ***@***.***>
wrote:

> I would like to start a discussion on the "Find & Select" pane. We are
> adding more and more information to it and I think the tabular view that we
> have right now no longer works. For inspiration, here are how other sites
> are doing it (data and journals):
>
> Figshare:
> [image: image]
> <https://user-images.githubusercontent.com/2530391/138554503-94448ab5-e3b5-444c-be31-d93657eb6d20.png>
>
> Zenodo:
> [image: image]
> <https://user-images.githubusercontent.com/2530391/138554525-071bb19f-f642-4581-a18f-58988204f86a.png>
>
> Springer:
> [image: image]
> <https://user-images.githubusercontent.com/2530391/138554541-bc817426-80a6-43be-bf59-da4607d43461.png>
>
> Elsevier:
> [image: image]
> <https://user-images.githubusercontent.com/2530391/138554581-7761641c-0e13-4ec3-b7a8-fe7a88faf449.png>
>
> I think I like the Zenodo way. I would fit quite well what we have. To the
> left of the author list I would have a little toggle button; if the toggle
> is off then the data is unpublished and we don't show authors (just the
> term "unpublished dataset" or similar).
>
> @tdbjacobs <https://github.com/tdbjacobs> @mcrot
> <https://github.com/mcrot> - any thoughts?
>
> This discussion is also related to issues #538
> <https://github.com/ContactEngineering/TopoBank/issues/538> #750
> <https://github.com/ContactEngineering/TopoBank/issues/750>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ContactEngineering/TopoBank/discussions/753>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFOUYWTUBI3LYFYRECM3GWTUIKOERANCNFSM5GSGBW4A>
> .
>
-- 
Tevis Jacobs
Mechanical Engineering and Materials Science
University of Pittsburgh

[Sent from my phone. Please forgive typos.]

**Comment by pastewka** [comment]
I agree, we don't need the graphics. I'll then work with @mcrot towards redesigning the search result view.

**Reply by mcrot** [reply]
I also like the Zenodo display, it is indeed very similar to all the information we already have.
The question is what to do with the collapsed measurements. Let's discuss further details in person.
Then we can post a suggestion here.

**Comment by pastewka** [comment]
I've transfere this to the issues
